content/node/staking.md

```diff
@@ -23,13 +23,13 @@ Prior to activation of non-custodial staking, this was the only option that node
 
 ## Non-custodial staking
 
-**Non-custodial staking is the process of having the operator of the node be a different account from the account that receives the rewards.**
+Non-custodial staking allows for the separation of the node operator and reward recipient accounts. There are two Pocket accounts involved in the process: the **operator** account and the **output** account.
 
-In this scenario, there are two Pocket accounts necessary to the staking of a node: **the operator account and the output account**.
+The operator account is responsible for signing claims and proofs, as well as block proposals and attestations. It needs to have some unstaked POKT to pay for these transactions.
 
-The operator account is the account from which the staked POKT is taken. The operator account also acts as the signer for claims and proofs, as well as block proposals and attestations, and so needs to have some POKT remaining in the account to pay for those transactions.
+The output account receives all rewards generated by the node. It also receives the original stake once the node is unstaked.
 
-The other account, known as the output account, is designated to receive the staked POKT when a node is unstaked, as well as receive all of the rewards generated by that node. The rewards generated by the node will be sent to the output address and will be available immediately. Only the initial stake itself is locked.
+Staking can be initiated by any account with sufficient POKT balance. The staked POKT will be taken from the account that initiates staking, and sent to the output account when unstaked.
```

Comment:
> Both the operator and output accounts can perform the staking and also any and all node maintenance (unstake, unjail, etc.). 
> 
> Non-custodial staking can be initiated by any operator or output account with sufficient POKT balance. The staked POKT will be taken from the account that initiates staking, and sent to the output account when unstaked.

Suggested light re-org of info and added in "non-custodial" in front of "...staking can be initiated..." just to be extra clear.